Week-2/Website/index.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>My Favourite</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <h1 class="row center">My Favourite Game!</h1>
+
+        <div class="row">
+            <p class="col">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dapibus ligula at nulla sollicitudin, ac efficitur libero posuere. Quisque vitae nisl suscipit, congue est non, convallis ex.</p>
+            <p class="col">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dapibus ligula at nulla sollicitudin, ac efficitur libero posuere. Quisque vitae nisl suscipit, congue est non, convallis ex.</p>
+        </div>
+
+        <div class="row">
+            <div class="col image">
```

Comment:
@mr-rego the text is responsive. But the expectation is the articles should come below image. Can you fix it ?

<img width="340" alt="Screenshot 2022-07-30 at 11 21 56 AM" src="https://user-images.githubusercontent.com/9133593/181876817-b626521f-3ae1-435e-86f1-dbfc5e0fc30d.png">